src/test/liquidity-mining/UniswapV3/MUNINewLogic.sol

```diff
@@ -0,0 +1,885 @@
+// SPDX-License-Identifier: UNLICENSED
```

Comment:
Do we really need a new file here? Would probably be easier to inherit from the original logic contract and add a new function (e.g. `function version() public returns (uint256) { returns 1; }`)), try and call that function and see if its successful?